### ISSUE TITLE
Добавление инкримента в инпут

### ISSUE THREAD

**Issue by AlexMetalcore** [unclead/yii2-multiple-input#324]
Доброй ночи!Возник вопрос! Можно ли средствами виджета сделать так что бы при клике на (+) (добвление) в поле https://prnt.sc/rhi0u6 добавлялся порядковый номер?

**Comment by unclead** [comment]
из коробки такого функционала нет, но можно реализовать самому используя  js события (в вики про них написано)

**Comment by AlexMetalcore** [comment]
> из коробки такого функционала нет, но можно реализовать самому используя js события (в вики про них написано)

Да, я уже после того как написал в вике увидил js события. Все ок, спасибо. Но есть еще один вопрос: как вывести данные с бд, что бы они так же корректно обновляли существующие и добавялил новые. Вот база https://prnt.sc/rhxbbo. Выводяться данные в зависимости от id домена. https://prnt.sc/rhxcwh - вот данные. Редактируеться только последняя запись 
Вот как в виде отображаеться

**Comment by unclead** [comment]
@AlexMetalcore  виджет не отвечает за логику сохранения, он лишь предоставляет функционал визуального управления. 

Рекомендую почитать https://github.com/unclead/yii2-multiple-input/wiki/Basic-scenario-of-usage
Если что-то не понятно, то лучше задать вопрос на официальном форуме yii  (https://yiiframework.ru/forum/)
